openmrs/apps/clinical/app.json

```diff
@@ -43,6 +43,9 @@
           "programAttributes": [
           ],
           "personAttributes": [
+            "givenNameLocal",
```

Comment:
>hence we should put Khmer name in widget.
So question is, should we keep the Dashboard widget for additional patient information then?

> It can be repositioned next to name if we change stuff in bahmniapps, Can we use git to include 2 repos for a commit?
Let's not change Bahmni Apps for now. Let's try to do with what we have.
No we cannot have a one commit across 2 repo.